lib/providers/kms.js

```diff
@@ -18,11 +18,24 @@ class KMSProvider {
     checkNotEmpty(secret, 'secret is required');
     checkNotEmpty(secret.ciphertext, 'secret.ciphertext is required');
 
+    let region = 'us-east-1';
+
+
+    if (secret.region && secret.region != null && secret.region !== '') { // eslint-disable-line eqeqeq
```

Comment:
I think you can write this as
```
if (secret.region && secret.region !== '') {
```
because `null` and `undefined` are both falsey.